src/style/blocks/search-form.css

```diff
@@ -0,0 +1,56 @@
+@import '../variables.css';
+
+.search-form {
+  display: flex;
+  box-sizing: border-box;
+  align-items: center;
+
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px var(--box-shadow-light-color);
+}
+
+.search-form--large {
+  height: 70px;
+}
+
+.search-form--small {
+  height: 42px;
+}
+
+.search-form--large:hover {
+  box-shadow: 0 4px 4px var(--box-shadow-dark-color);
+}
+
+.search-form--small:hover {
+  box-shadow: 0 3px 8px var(--box-shadow-darker-color);
+}
```

Comment:
you can use it for general class. do not duplicate

Reply:
I don't quite understand. How do I combine them if they have different styles?